biomedical_schema/encode.mcf

```diff
@@ -1,902 +1,1187 @@
-Node: encode_class_BedLine
+Node: BedLine
```

Comment:
Thanks for the schema updates.  Is it the case that data with this updated schema already exists?

Reply:
The schema was lacking the rangeIncludes and domainIncludes and therefore didn't link properly to itself within the browser. I've added this information for each node and property.

Reply:
Ah, got it. I thought you were changing the DCID, but you were only changing the node name (like encode_property_fromBedFile), so this doesn't impact data!